PopPUNK/refine.py

```diff
@@ -30,7 +30,6 @@
     import cudf
     gpu_lib = True
 except ImportError as e:
-    sys.stderr.write("cugraph and cudf unavailable\n")
```

Comment:
Printing this error seems like it would still be useful in one file

Reply:
Restored in 5b362fb.